modules/ncc-spoke-ra/variables.tf

```diff
@@ -21,12 +21,17 @@ variable "data_transfer" {
 }
 
 variable "hub" {
-  description = "The name of the NCC hub to create or use."
+  description = "The name of the NCC hub to create or the id for an existing hub to use."
```

Comment:
Should we fix a bit this description? It seems we're passing an object rather than a string.

Example:
The NCC hub. You should either provide an existing hub id or a hub name if create is true.

Reply:
Updated description as suggested!